pdns/recursordist/rec-main.cc

```diff
@@ -1891,31 +1891,31 @@ static unsigned int initDistribution(Logr::log_t log)
       for (unsigned int i = 0; i < RecThreadInfo::numDistributors(); i++, threadNum++) {
         auto& info = RecThreadInfo::info(threadNum);
         auto& deferredAdds = info.getDeferredAdds();
-        count += makeUDPServerSockets(deferredAdds, log);
+        count += makeUDPServerSockets(deferredAdds, log, i == RecThreadInfo::numDistributors() - 1, RecThreadInfo::numDistributors());
```

Comment:
would `i == 0` perhaps be more readable?

Reply:
it's also lightly confusing that you pass an instance count but the called function does not use it (except for logging), but I see how that would be a bigger rework.

Maybe logging just below this for is cleaner?

Reply:
My idea was to log about socket creation after all has been done, hence the end condition. Let me take a look into logging after the creation.

Reply:
> hence the end condition.

That did occur to me :)

Reply:
right. maybe a comment somewhere that explains that the argument is not entirely functional then